intel-retail/automated-self-checkout
# refactor!: remove --workload parameter from run.sh and deprected --workload dlstreamer run

BREAKING CHANGE: we no longer support --workload parameter from run.sh and thus deprected the dlstreamer workload use cases.

  Closes: #381

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**381**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- make clean-all
- test model downloads: `make clean-models; ./download_models/getModels.sh`  should see models downloaded without problem
- unit tests: `make unit-test-profile-launcher` should see all passed
- functional test: you can run any pipeline profile without using **--workload** and it should work; one example:
```console
$ make run-camera-simulator
$ PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
```
- smoke test: `make run-smoke-tests` should see ALL PASSED
- 

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
